### ISSUE TITLE
[BUG] Let's Encrypt not working when MULTISITE=no

### ISSUE THREAD

**Issue by fl0ppy-d1sk** [bunkerity/bunkerweb#279]
**Description**
Certificate generation of Let's Encrypt is not working when `MULTISITE` is set to `no`.

**How to reproduce**
```shell
docker volume create bw-data && \
docker network create bw-services && \
docker run -d \
       --name mybunker \
       --network bw-services \
       -p 80:8080 \
       -p 443:8443 \
       -v bw-data:/data \
       -e SERVER_NAME=admin2.domain.com \
       -e AUTO_LETS_ENCRYPT=yes \
       -e USE_REVERSE_PROXY=yes \
       -e REVERSE_PROXY_KEEPALIVE=yes \
       -e REVERSE_PROXY_URL=/ \
       -e REVERSE_PROXY_HOST=http://192.168.7.189:8069 \
       bunkerity/bunkerweb:1.4.2
```

**Logs**
```logs
...
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Account registered.
Requesting a certificate for admin2.domain.com
 
Certbot failed to authenticate some domains (authenticator: manual). The Certificate Authority reported these problems:
  Domain: admin2.domain.com
  Type:   connection
  Detail: xxx.xxx.xx.xx: Fetching http://admin2.domain.com/.well-known/acme-challenge/rduIE37dKO2SfiPLPaSqqKd7KkZ4o5fJW9mRrOEwWqU: Connection refused
 
Hint: The Certificate Authority failed to verify the challenge files created by the --manual-auth-hook. Ensure that this hook is functioning correctly. Refer to "certbot --help manual" and the Certbot User Guide.
...
```

**Comment by makorne** [comment]
Looks like LetsEncrypt automatic certificate generation and renewal do not work on Linux with MULTISITE=no or yes too.

```
● bunkerweb.service - BunkerWeb service
   Loaded: loaded (/etc/systemd/system/bunkerweb.service; enabled; vendor preset: disabled)
   Active: active (running) since Thu 2022-08-11 05:31:04 MSK; 3s ago
     Docs: https://docs.bunkerweb.io
  Process: 1809066 ExecStop=/opt/bunkerweb/scripts/start.sh stop (code=exited, status=0/SUCCESS)
 Main PID: 1809071 (start.sh)
    Tasks: 5 (limit: 22686)
   Memory: 13.2M
   CGroup: /system.slice/bunkerweb.service
           ├─1809071 /bin/bash /opt/bunkerweb/scripts/start.sh start
           ├─1809076 nginx: master process nginx
           ├─1809077 nginx: worker process
           ├─1809078 nginx: worker process
           └─1809094 nginx -s quit

Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] WHITELIST - ℹ️ - Whitelist for IP is not cached, processing downloads...
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] WHITELIST - ℹ️ - Whitelist for RDNS is not cached, processing downloads...
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] WHITELIST - ℹ️ - Whitelist for ASN is not cached, processing downloads...
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] WHITELIST - ℹ️ - Whitelist for USER_AGENT is not cached, processing downloads...
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] WHITELIST - ℹ️ - Whitelist for URI is not cached, processing downloads...
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] SCHEDULER - ℹ️ - Successfuly executed job whitelist-download from plugin whitelist
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] SCHEDULER - ℹ️ - All jobs in run_once() were successful
Aug 11 05:31:07 Hilbert start.sh[1809071]: [2022-08-11 05:31:07] SCHEDULER - ℹ️ - Job scheduler stopped
Aug 11 05:31:07 Hilbert start.sh[1809071]: Run jobs script
Aug 11 05:31:07 Hilbert start.sh[1809071]: OK !
```
Successfuly executed job certbot-renew from plugin letsencrypt
but  certificates are still expired

```
Aug 11 05:41:47 start.sh[1809207]: [2022-08-11 05:41:47] SCHEDULER - ℹ️ - Successfuly executed job custom-cert from plugin customcert
Aug 11 05:41:47 start.sh[1809207]: [2022-08-11 05:41:47] SCHEDULER - ℹ️ - Executing job mmdb-country from plugin jobs ...
Aug 11 05:41:47 start.sh[1809207]: [2022-08-11 05:41:47] JOBS - ℹ️ - country.mmdb is already in cache, skipping download...
Aug 11 05:41:47 start.sh[1809207]: [2022-08-11 05:41:47] SCHEDULER - ℹ️ - Successfuly executed job mmdb-country from plugin jobs
Aug 11 05:41:47 start.sh[1809207]: [2022-08-11 05:41:47] SCHEDULER - ℹ️ - Executing job mmdb-asn from plugin jobs ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] JOBS - ℹ️ - asn.mmdb is already in cache, skipping download...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job mmdb-asn from plugin jobs
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job download-plugins from plugin jobs ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] JOBS - ℹ️ - No external plugins to download
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job download-plugins from plugin jobs
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job certbot-new from plugin letsencrypt ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job certbot-new from plugin letsencrypt
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job certbot-renew from plugin letsencrypt ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job certbot-renew from plugin letsencrypt
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job realip-download from plugin realip ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] REALIP - ℹ️ - RealIP is not activated, skipping download...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job realip-download from plugin realip
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job self-signed from plugin selfsigned ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job self-signed from plugin selfsigned
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job whitelist-download from plugin whitelist ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] WHITELIST - ℹ️ - Whitelist for IP is not cached, processing downloads...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] WHITELIST - ℹ️ - Whitelist for RDNS is not cached, processing downloads...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] WHITELIST - ℹ️ - Whitelist for ASN is not cached, processing downloads...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] WHITELIST - ℹ️ - Whitelist for USER_AGENT is not cached, processing downloads...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] WHITELIST - ℹ️ - Whitelist for URI is not cached, processing downloads...
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] SCHEDULER - ℹ️ - Successfuly executed job whitelist-download from plugin whitelist
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] SCHEDULER - ℹ️ - All jobs in run_once() were successful
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] SCHEDULER - ℹ️ - Job scheduler stopped
Aug 11 05:41:49 start.sh[1809207]: Run jobs script
Aug 11 05:41:49 start.sh[1809207]: OK !
Aug 11 05:41:49 start.sh[1809207]: Quit nginx
Aug 11 05:41:49 start.sh[1809207]: OK !
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] GENERATOR - ℹ️ - Generator started ...
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] GENERATOR - ℹ️ - Settings : /opt/bunkerweb/settings.json
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] GENERATOR - ℹ️ - Templates : /opt/bunkerweb/confs
Aug 11 05:41:49 start.sh[1809207]: [2022-08-11 05:41:49] GENERATOR - ℹ️ - Core : /opt/bunkerweb/core
```
I tried AUTO_LETS_ENCRYPT no/yes  and   MULTISITE=no/yes
with restart bunkerweb 

Maybe there is a command for manual renewal of them?

**Comment by makorne** [comment]
I found an error in the old log:

```
Aug  8 17:50:49  start.sh[1766040]: [2022-08-08 17:50:49] SCHEDULER - ℹ️ - Executing job certbot-renew from plugin letsencrypt ...
Aug  8 17:50:49  start.sh[1766040]: Traceback (most recent call last):
Aug  8 17:50:49  start.sh[1766040]:  File "/opt/bunkerweb/deps/python/bin/certbot", line 5, in <module>
Aug  8 17:50:49  start.sh[1766040]:    from certbot.main import main
Aug  8 17:50:49  start.sh[1766040]:  File "/opt/bunkerweb/deps/python/certbot/main.py", line 6, in <module>
Aug  8 17:50:49  start.sh[1766040]:    from certbot._internal import main as internal_main
Aug  8 17:50:49  start.sh[1766040]:  File "/opt/bunkerweb/deps/python/certbot/_internal/main.py", line 19, in <module>
Aug  8 17:50:49  start.sh[1766040]:    import zope.component
Aug  8 17:50:49  start.sh[1766040]:  File "/opt/bunkerweb/deps/python/zope/component/__init__.py", line 25, in <module>
Aug  8 17:50:49  start.sh[1766040]:    from zope.interface import Interface
Aug  8 17:50:49  start.sh[1766040]: ModuleNotFoundError: No module named 'zope.interface'
Aug  8 17:50:49  start.sh[1766040]: [2022-08-08 17:50:49] LETS-ENCRYPT - ❌ - Certificates renewal for  failed
Aug  8 17:50:49  start.sh[1766040]: [2022-08-08 17:50:49] SCHEDULER - ❌ - Error while executing job certbot-renew from plugin letsencrypt
```
But no errors later and certificates are not renewed 

```
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Executing job certbot-renew from plugin letsencrypt ...
Aug 11 05:41:48 start.sh[1809207]: [2022-08-11 05:41:48] SCHEDULER - ℹ️ - Successfuly executed job certbot-renew from plugin letsencrypt
```
What do I do?

**Comment by makorne** [comment]
Manual renewal gave the error:

```
Hook 'deploy-hook' reported error code 1
Hook 'deploy-hook' ran with output:
 [2022-08-15 09:15:49] LETS-ENCRYPT - ❌ - Exception while running certbot-auth.py :
 Traceback (most recent call last):
   File "/opt/bunkerweb/core/letsencrypt/jobs/certbot-auth.py", line 45, in <module>
     with open(root_dir + token, "w") as f :
 TypeError: must be str, not NoneType

```
But certificate got renewed

**Comment by fl0ppy-d1sk** [comment]
Hello @makorne, it should be fixed with the last v1.4.3 release.
